### PR TITLE
Cristhian

### PR DESCRIPTION
**Pull request by CristhianRF24**
I added the listing to the applications page and moved some of the logic from the pagination-section because it gave me problems implementing pagination functionality, but it doesn't affect the other paginations.

### CODE REVIEW

File: src/app/startups/applications/page.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    });
+  }, []);
+
+  const totalPages = Math.ceil(data.length / ITEMS_PAGE);
+  const paginatedData = data.slice(
+    (currentPage - 1) * ITEMS_PAGE,
+    currentPage * ITEMS_PAGE
+  );
+
+//  console.log(`current: ${currentPage}, paginate: ${paginatedData.length} items, total: ${totalPages} pages`);
```
Comment (MartinArispeRiveros) [inline src/app/startups/applications/page.tsx:32]: @CristhianRF24 please remove commented code

File: src/components/table-section-applications.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+interface TableSectionAplicationsProps {
+    
+    data: TableApplicationsPorps[];
+} 
+
+
+
+export default function TableSectionAplications({ data }: TableSectionAplicationsProps) {
+    return (
+        <div className="mb-4 ml-8 mr-8 mt-5 p-4 border border-black rounded-lg shadow-[0px_5px_5px_rgba(0,0,0,0.5)] overflow-hidden">
```
Comment (MartinArispeRiveros) [inline src/components/table-section-applications.tsx:22]: @CristhianRF24 please add shadow to css section

File: src/components/table-section-applications.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    phone: string;
+   
+}
+
+interface TableSectionAplicationsProps {
+    
+    data: TableApplicationsPorps[];
+} 
+
+
```
Comment (MartinArispeRiveros) [inline src/components/table-section-applications.tsx:18]: @CristhianRF24 please remove white lines

File: src/components/table-application.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+                    <Typography tag="p" alignment="left" fontWeight="bold">
+                        {startup_name}
+                    </Typography>
+                    <Typography tag="p" alignment="left">
+                        {full_name} {" - "}
+                        {email}  {" - "}
+                        {phone}  
+                    </Typography>
+                    
+
```
Comment (MartinArispeRiveros) [inline src/components/table-application.tsx:28]: @CristhianRF24 remove white lines

File: src/components/table-application.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+import InformationButton from "./information-buton";
+import Typography from "./Typography/typography";
+import { TableRow, TableCell } from "./ui/table";
+
+interface TableApplicationsPorps {
+    startup_name: string;
+    full_name: string;
+    email: string;
+    phone: string;
+
```
Comment (MartinArispeRiveros) [inline src/components/table-application.tsx:11]: @CristhianRF24 please remove white lines

File: src/components/side-menu.tsx [hunk 6]
```diff
@@ -42,7 +42,7 @@ export default function SideMenu() {
                                         <AccordionContent className="pl-8">
                                             <ul className="space-y-2">
                                                 <li><a href="/startups" className="block p-2 rounded-md hover:bg-[#F2A594] " >Lista de Startups</a></li>
-                                                <li><a href="/startups/postulaciones" className="block p-2 rounded-md hover:bg-[#F2A594]">Lista de Postulaciones</a></li>
+                                                <li><a href="/startups/applications" className="block p-2 rounded-md hover:bg-[#F2A594]">Lista de Postulaciones</a></li>
```
Comment (MartinArispeRiveros) [inline src/components/side-menu.tsx:45]: @CristhianRF24 please assign colors into css section

File: src/components/pagination-section.tsx [hunk 7]
```diff
@@ -13,27 +13,94 @@ export default function PaginationSection({ currentPage, totalPages, onPageChang
...
                     </PaginationItem>
                 ))}
+
+               
+                {showEllipsisRight && (
+                    <>
+                        <PaginationEllipsis />
+                        <PaginationItem>
+                            <PaginationLink
+                                href="#"
```
Comment (MartinArispeRiveros) [inline src/components/pagination-section.tsx:79]: @CristhianRF24 please remove href parameter if you don't use it

File: src/components/pagination-section.tsx [hunk 8]
```diff
@@ -13,27 +13,94 @@ export default function PaginationSection({ currentPage, totalPages, onPageChang
...
+                        </PaginationItem>
+                        <PaginationEllipsis />
+                    </>
+                )}
+
+                {Array.from({ length: endPage - startPage + 1 }, (_, i) => startPage + i).map(page => (
                     <PaginationItem key={page}>
-                        <PaginationLink href="#" isActive={page === currentPage} onClick={() => handlePageChange(page)}>
+                        <PaginationLink
+                            href="#"
```
Comment (MartinArispeRiveros) [inline src/components/pagination-section.tsx:61]: @CristhianRF24 please remove href parameter if you don't use it

File: src/components/pagination-section.tsx [hunk 9]
```diff
@@ -13,27 +13,94 @@ export default function PaginationSection({ currentPage, totalPages, onPageChang
...
                     </PaginationItem>
                 )}
-                {Array.from({ length: totalPages }, (_, i) => i + 1).map(page => (
+
+               
+                {showEllipsisLeft && (
+                    <>
+                        <PaginationItem>
+                            <PaginationLink
+                                href="#"
```
Comment (MartinArispeRiveros) [inline src/components/pagination-section.tsx:45]: @CristhianRF24 please remove href parameter if you don't use it

File: src/components/pagination-section.tsx [hunk 10]
```diff
@@ -13,27 +13,94 @@ export default function PaginationSection({ currentPage, totalPages, onPageChang
...
+
     return (
         <Pagination>
             <PaginationContent>
+                {/* Previous Button */}
                 {currentPage > 1 && (
                     <PaginationItem>
-                        <PaginationPrevious href="#" onClick={(event) => { event.preventDefault(); handlePageChange(currentPage - 1); }} />
+                        <PaginationPrevious
+                            href="#"
```
Comment (MartinArispeRiveros) [inline src/components/pagination-section.tsx:31]: @CristhianRF24 please remove href parameter if you don't use it

File: src/components/delete-button.tsx [hunk 11]
```diff
@@ -5,6 +5,7 @@ export default function DeleteButton() {
     return (
         <Button className="w-10 h-10 p-2 bg-red-600 text-white rounded-full flex items-center justify-center hover:bg-[#FE420A]">
             <TrashIcon className="w-11 h-11" />
+
```
Comment (MartinArispeRiveros) [inline src/components/delete-button.tsx:8]: @CristhianRF24 remove extra white line

File: src/app/startups/applications/page.tsx [hunk 12]
```diff
@@ -0,0 +1,50 @@
+"use client";
+import { database } from "@/app/firebaseConfig";
+import PaginationSection from "@/components/pagination-section";
+import SearchCard from "@/components/search-card";
+import TableSectionAplications from "@/components/table-section-applications";
+import Title from "@/components/title";
+import { ref, onValue } from "firebase/database";
+import { useState, useEffect } from "react";
+
+const ITEMS_PAGE = 6; // Mostrar 6 elementos por página
```
Comment (MartinArispeRiveros) [inline src/app/startups/applications/page.tsx:10]: @CristhianRF24 please comments in English

**Review by MartinArispeRiveros — CHANGES_REQUESTED** [review]

File: src/app/startups/applications/page.tsx (hunk 12 above)
Comment (richi0085) [reply src/app/startups/applications/page.tsx:10]: It looks a chatGPT code 
I have no problem with it but please , read the code not just copy and paste

**Pull request closed without merging**